simple.js

```diff
@@ -1313,8 +1313,9 @@ define(function(require, exports, module) {
             // add Gravatar toggle online only
             info.getUser(addGravatarToggle);
 
-            // add C template
+            // add templates
             newresource.addFileTemplate(require("text!./templates/c.templates"), plugin);
+            newresource.addFileTemplate(require("text!./templates/py.templates"), plugin);
```

Comment:
Since newresource doesn't support removing template, we just need to remove `File/New From Template/Python file` before [L1318](https://github.com/cs50/harvard.cs50.simple/pull/65#discussion-diff-97089089R1318).

A couple of other problems I just noticed are
 1. there needs to be a tab in each blank line inside the template
 2. terminating newline needs to be removed (triggers adding a blank template)

(Let me know if I should address, if busy!)